src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.List;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(Fruit fruit) {
+        Storage.fruits.add(fruit);
```

Comment:
Direct manipulation of storage within DAO methods can lead to problems. Instead, consider providing methods within the Storage class to handle these operations, ensuring encapsulation and single responsibility principles are followed.